lib/Resources/config/search/common/layouts_page_text_indexing.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+    netgen.ibexa_search_extra.page_indexing.page_text_extractor:
+        class: Netgen\IbexaSearchExtra\Core\Search\Common\PageTextExtractor
+        arguments:
+            - '@Ibexa\Contracts\Core\Persistence\Content\Handler'
+            - '@Ibexa\Contracts\Core\Persistence\Content\Location\Handler'
+            - '@Ibexa\Bundle\Core\Routing\UrlAliasRouter'
+            - '%netgen_ibexa_search_extra.page_indexing.site_roots%'
+            - '%netgen_ibexa_search_extra.page_indexing.languages_siteaccess_map%'
+            - '%netgen_ibexa_search_extra.page_indexing.host%'
+            - '%netgen_ibexa_search_extra.page_indexing.config%'
+        calls:
+            - setLogger: ['@?logger']
+
+    netgen.ibexa_search_extra.page_indexing:
+        class: Netgen\IbexaSearchExtra\Command\IndexPageContentCommand
+        tags:
+            - { name: console.command }
+        arguments:
+            - '@Ibexa\Contracts\Core\Repository\ContentService'
+            - '@ibexa.spi.search'
+            - '@ibexa.api.persistence_handler'
+            - '%netgen_ibexa_search_extra.page_indexing.allowed_content_types%'
+
+#parameters:
```

Comment:
This should be removed or uncommented.

Reply:
resolved in 497a329387786154c853232148c58bd86e3e230c